charts/portworx/templates/portworx-api.yaml

```diff
@@ -0,0 +1,86 @@
+{{- if (.Values.api.enabled ) }}
```

Comment:
the portworx-api manifests have since been added to the chart as part of https://github.com/portworx/helm/pull/123. So this file can be removed.